recipe/meta.yaml

```diff
@@ -62,34 +76,34 @@ outputs:
 
   - name: libmambapy
     version: {{ libmambapy_version }}
-    script: build_mamba.sh                       # [unix]
-    script: build_mamba.bat                      # [win]
+    script: build_libmambapy.sh                   # [unix]
+    script: bld_libmamba.bat                      # [win]
     build:
       string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}
       run_exports:
         - {{ pin_subpackage('libmambapy', max_pin='x.x') }}
+      missing_dso_whitelist:  # [s390x]
+        - $RPATH/ld64.so.1    # [s390x] Know s390x `ld64.so` issue.
       ignore_run_exports:
         - spdlog
     requirements:
       build:
         - {{ compiler('cxx') }}
         - cmake
         - ninja
-        - python                                 # [build_platform != target_platform]
-        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-        - pybind11                               # [build_platform != target_platform]
-        - pybind11-abi                           # [build_platform != target_platform]
       host:
         - python
         - pip
-        - pybind11
-        - pybind11-abi
-        - openssl
-        - yaml-cpp
-        - spdlog
+        - libsolv >=0.7.19
+        - pybind11 >=2.2
+        #- openssl
+        #- yaml-cpp
+        #- spdlog-fmt-embed
         - cpp-filesystem >=1.5.8
         - termcolor-cpp
         - nlohmann_json
+        - cli11
+        - winreg                                 # [win]
         - {{ pin_subpackage('libmamba', exact=True) }}
```

Comment:
There are missing host dependencies `setuptools` and `wheel`

Reply:
Added.